arrow-fx/src/main/kotlin/arrow/fx/IOParMap.kt

```diff
@@ -7,60 +7,259 @@ import arrow.core.Option
 import arrow.core.Right
 import arrow.core.Tuple2
 import arrow.core.Tuple3
+import arrow.core.Tuple4
+import arrow.core.Tuple5
+import arrow.core.Tuple6
+import arrow.core.Tuple7
+import arrow.core.Tuple8
+import arrow.core.Tuple9
 import arrow.core.extensions.option.applicative.applicative
+import arrow.core.extensions.option.applicativeError.handleError
+import arrow.core.internal.AtomicBooleanW
 import arrow.core.internal.AtomicRefW
 import arrow.core.nonFatalOrThrow
 import arrow.core.none
 import arrow.core.some
+import arrow.core.toT
 import arrow.fx.internal.IOForkedStart
 import arrow.fx.internal.Platform
 import kotlin.coroutines.CoroutineContext
-import arrow.core.extensions.option.applicativeError.handleError
-import arrow.core.internal.AtomicBooleanW
 
-/** Mix-in to enable `parMapN` 2-arity on IO's companion directly. */
+/** Mix-in to enable `parMapN` N-arity on IO's companion directly. */
 interface IOParMap {
 
+  @Deprecated("This API is not consistent with others within Arrow, see version with Tuple instead of function params")
```

Comment:
No need for `ReplaceWith` here?

Reply:
Unfortunately it cannot be autocorrected AFAIK, but the signature is almost identical